### PR TITLE
adding info about policy dependencies

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/stolostron/backlog/issues/27004

### CODE REVIEW

File: governance/policy_dependencies.adoc [hunk 1]
```diff
@@ -0,0 +1,59 @@
+[#policy-dependencies]
+= Policy dependencies
+
+Dependencies can be used to activate policy or policy template tasks when another policy is in a certain state. The following fields are checked on the managed cluster, `dependencies` and `extraDependencies`. When a dependency is not met, the template status of the replicated policy template displays more details.
```
Comment (gparvin) [inline governance/policy_dependencies.adoc]: The phrase "activate policy or policy template tasks" makes it sound like there's a task concept being introduced here which is not the case.  I think the definition should be clarified a little more to make it clear it's the policy or policy template itself that is activated through passing the dependency criteria.
Thanks!

**Comment by dockerymick** [comment]
@gparvin thanks Gus!

**Comment by gparvin** [comment]
Thanks Mikela, It looks good to me.  I'll let Justin review too.

**Review by swopebe — APPROVED** [review]
beautiful

File: governance/policy_dependencies.adoc [hunk 2]
```diff
@@ -0,0 +1,59 @@
+[#policy-dependencies]
+= Policy dependencies
+
+Dependencies can be used to activate a policy or policy template through passing the dependency criteria. The following fields are checked on the managed cluster, `dependencies` and `extraDependencies`. When a dependency is not met, the template status of the replicated policy template displays more details.
```
Comment (JustinKuli) [inline governance/policy_dependencies.adoc]:
```suggestion
Dependencies can be used to activate a policy or policy template when the dependency criteria are satisfied. The following fields are checked on the managed cluster, `dependencies` and `extraDependencies`. When a dependency is not met, the template status of the replicated policy template displays more details.
```

File: governance/policy_dependencies.adoc [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+              name: ocp4-moderate-node
+            settingsRef:
+              apiGroup: compliance.openshift.io/v1alpha1
+              kind: ScanSetting
+              name: default
+        remediationAction: enforce
+        severity: low
+----
+
+*Note:* A dependency cannot be used to apply a policy on one cluster based on the status of another.
```
Comment (JustinKuli) [inline governance/policy_dependencies.adoc]:
```suggestion
*Note:* A dependency cannot be used to apply a policy on one cluster based on the status of a policy in another cluster. 
```

File: governance/policy_intro.adoc [hunk 4]
```diff
@@ -42,6 +42,13 @@ metadata:
     policy.open-cluster-management.io/categories:
     policy.open-cluster-management.io/controls:
 spec:
+  dependencies:
+  - apiVersion: policy.open-cluster-management.io/v1
+    compliance:
+    kind: Policy
+    name: 
+    namespace: default
+  disabled: false
```
Comment (JustinKuli) [inline governance/policy_intro.adoc]: `disabled` ended up in this YAML twice, and I think that the `namespace` field could be left blank to match the other fields in this YAML

**Review by JustinKuli — COMMENTED** [review]
Looks great, just a couple of small things!

**Comment by dockerymick** [comment]
@JustinKuli @gparvin @swopebe thanks for the reviews and approval! I have accepted Justin's suggestions and made updates related to his feedback. @JustinKuli will you provide approval?

**Review by JustinKuli — APPROVED** [review]
/lgtm

**Pull request merged**